### PR TITLE
Update UG Images for visual aid

### LINKED ISSUE #298: [PE-D][Tester A] Insufficient Visual Aid In User Guide
More images of the UI might be useful for users' understanding, especially for commands that are related to orders, since those depend on the displayed list of orders such as markorder

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: NicholasT1/ped#8

### LINKED ISSUE #302: [PE-D][Tester D] Standardization of image cropping
The cropping of images are not consistent in UG.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: zyng233/ped#16

### PR DESCRIPTION
**Pull request by kaixuan477**
Close #298 
Close #302

### CODE REVIEW

**Review by awdse22 — COMMENTED** [review]
LGTM but what about for adding customer, listing customer, etc.?
If we're adding addorder, listorder, markorder, filterorder, then wouldn't it make sense to just do for everything?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -214,6 +214,7 @@ Examples:
 * `addorder 2 i/Lamp d/20-11-2024 q/3` adds the order with item `Lamp`, quantity of **3** and delivery date `20-11-2024`, to the 2nd customer in the address book.
 * `addo 1 i/Books d/02-03-2026` adds the order with item `Books`, quantity of **1** and delivery date `02-03-2026`, to the first customer in the address book.
 * `findcustomer Betsy` followed by `addorder 1 i/Bottles d/12-12-2002 q/1` adds an order under the 1st customer in the results of the `findcustomer` command.
+  ![result for adding order](images/addOrder.png)
```
Comment (FooChao) [inline docs/UserGuide.md:217]: this image does not match that of betsy

File: docs/UserGuide.md [hunk 2]
```diff
@@ -240,7 +241,7 @@ Format: `listorder INDEX`<br>
 Examples:
 * `listcustomer` followed by `listorder 2` lists all orders of the 2nd customer in the address book.
 * `findcustomer Betsy` followed by `listo 1` lists all orders of the 1st customer from the list of customers found with "Betsy".
-
+  ![result for 'listo 1'](images/listOrder.png)
```
Comment (FooChao) [inline docs/UserGuide.md:244]: this image does not match that of betsy order as well

File: docs/UserGuide.md [hunk 3]
```diff
@@ -305,6 +306,7 @@ Format: `markorder ORDER_INDEX`<br>
 Examples:
 * `markorder 1` marks the order with index 1 from the displayed order list as completed.
 * `listorder 1` followed by `marko 2` selects the 1st customer in the address book and marks the 2nd order under the 1st customer as completed.
+  ![result for 'marko 1'](images/markOrder.png)
```
Comment (FooChao) [inline docs/UserGuide.md:309]: well this image is below marko 2...

**Review by FooChao — COMMENTED** [review]
Many of your image does not match what is said. Please change.

File: docs/UserGuide.md (hunk 1 above)
Comment (kaixuan477) [reply docs/UserGuide.md:217]: Updated in commit.

File: docs/UserGuide.md (hunk 2 above)
Comment (kaixuan477) [reply docs/UserGuide.md:244]: Updated in commit.

File: docs/UserGuide.md (hunk 3 above)
Comment (kaixuan477) [reply docs/UserGuide.md:309]: Updated in commit.

**Review by FooChao — APPROVED** [review]
LGTM

**Pull request merged**